### PR TITLE
Interp fixes

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
build-interpreter

**Comment by lewurm** [comment]
build interpreter

File: mono/mini/interp/transform.c [hunk 1]
```diff
@@ -820,9 +820,10 @@ interp_transform_call (TransformData *td, MonoMethod *method, MonoMethod *target
 			if (mheader && *mheader->code == CEE_RET && called_inited) {
 				if (mono_interp_traceopt)
 					g_print ("Inline (empty) call of %s.%s\n", target_method->klass->name, target_method->name);
-				for (i = 0; i < csignature->param_count; i++)
+				for (i = 0; i < csignature->param_count; i++) {
 					ADD_CODE (td, MINT_POP); /*FIX: vt */
 					ADD_CODE (td, 0);
+				}
```
Comment (lewurm) [inline mono/mini/interp/transform.c]: :( good catch

File: mono/mini/mini-runtime.c [hunk 2]
```diff
@@ -586,20 +586,22 @@ mono_debug_count (void)
 	static int count = 0;
 	static gboolean inited;
 	static char *value;
+	static int int_val = -0xffff;
 
 	count ++;
 
 	if (!inited) {
 		value = g_getenv ("COUNT");
+		if (value) {
```
Comment (kumpera) [inline mono/mini/mini-runtime.c]: This still wrong, value must not be static.

Comment (kumpera) [inline mono/mini/mini-runtime.c]: I got that fixed in another PR, so don't sweat about making the change.

**Pull request merged**